### ISSUE TITLE
Wiki Compliance Script typo

### ISSUE THREAD

**Issue by its-a-lisa** [usnistgov/macos_security#293]
### Summary

The [Compliance Script](https://github.com/usnistgov/macos_security/wiki/Compliance-Script ) entry for the Wiki has a bug on the last step

### Steps to reproduce

Follow the instructions:

1. ./scripts/generate_guidance.py -s baselines/800-53r5_moderate.yaml
Profile YAML: baselines/800-53r5_moderate.yaml
2. ls -dn build/800-53r5_moderate/*
3. sudo ./build/800-53_moderate/800-53r5_moderate_compliance.sh

### Operating System version

Not applicable 

### Intel or Apple Silicon

Not applicable 

### What is the current *bug* behavior?

 If you follow all the steps and then try to run `sudo ./build/800-53_moderate/800-53r5_moderate_compliance.sh` you get a result of `sudo: ./build/800-53_moderate/800-53r5_moderate_compliance.sh: command not found`

### What is the expected *correct* behavior?

- You should be prompted to enter in your password and then be presented with the Main-Menu screen

### Relevant logs and/or screenshots

Not applicable 

### Output of checks

Not applicable 

### Possible fixes

TL;DR - The directory is wrong and missing "r5" so it should really be /build/800-53**r5**_moderate/

To achieve this you need to update the command to `sudo ./build/800-53r5_moderate/800-53r5_moderate_compliance.sh`

**Comment by brodjieski** [comment]
Ahh!  Thanks for finding the typo!  I've updated the wiki to reflect the correct syntax.
